src/custom-element.js

```diff
@@ -566,6 +566,11 @@ function createBaseCustomElementClass(win) {
 
       /** @private {?./layout-delay-meter.LayoutDelayMeter} */
       this.layoutDelayMeter_ = null;
+
+      if (this.whenUpgradeToCustomElement_) {
```

Comment:
Our current format is smth like `__AMP_NAME_OF_FIELD`. Let's keep to that format.